### DISCUSSION TITLE
Assets not seen after adding all networks!

### DISCUSSION THREAD

**Discussion by amazingHuo** [OneKeyHQ/app-monorepo#3194] [General]
![image](https://github.com/OneKeyHQ/app-monorepo/assets/17975200/bbddc0d4-cfa0-42d4-a1ae-9529a49b9f0f)

**Comment by OneWalletToRuleThemAll** [comment]
I am still having this issue. Some tokens value will appear and some will not when selecting all accounts.

**Comment by revan-zhang** [comment]
You can upgrade to v5.x version and try again. We have designed a new mode for asset loading.
